modules/javafx.graphics/src/main/java/javafx/scene/Scene.java

```diff
@@ -1175,6 +1187,155 @@ public String getName() {
         return fill;
     }
 
+    /**
+     * Specifies whether the scene should prefer light text on dark backgrounds, or dark text
+     * on light backgrounds.
+     * <p>
+     * This is a <em>null-coalescing</em> property: if set to {@code null}, it evaluates to the
+     * value of {@link Platform.Preferences#colorSchemeProperty()}. Therefore, specifying a value
+     * for this property overrides the platform-provided value.
+     * <p>
+     * This property corresponds to the following CSS media feature:
+     * <table class="striped">
+     *     <caption>Media Feature</caption>
+     *     <tbody>
+     *         <tr><th>Name</th><td><code>prefers-color-scheme</code></td></tr>
+     *         <tr><th>For</th><td><code>@media</code></td></tr>
+     *         <tr><th>Value</th><td><code>light</code> | <code>dark</code></td></tr>
+     *         <tr><th>Boolean Context</th><td>no</td></tr>
+     *     </tbody>
+     * </table>
+     *
+     * @defaultValue {@link Platform.Preferences#getColorScheme()}
+     * @since 24
+     */
+    private final ObjectProperty<ColorScheme> colorScheme = new MediaProperty<>(
```

Comment:
is it possible for two different scenes to have different color schemes or any other properties derived from the platform preferences?

would it make more sense to use the platform preferences properties directly?